index.html

```diff
@@ -22,5 +22,8 @@ <h1 class="header-left">
             </a>
         </div>
     </header>
+    <footer class="site-footer">
+        <p>© 2025 有限会社ヤタカ物産 All rights reserved.</p>
```

Comment:
@K-YYY 
こちら現場ではこんな風にコーディングするので、修正お願いできればです！🙏

- コピーライトなどはsmallタグでマークアップ
- ©などの特殊文字は（他には商標記号など）、ブラウザによっては正しく表示されない場合があるので `&copy;` という実体参照で書きます

```suggestion
        <p><small>&copy; 2025 有限会社ヤタカ物産 All rights reserved.</small></p>
```

実体参照については、記事を貼っておくので、この機会に見て、知ってもらえると👍
使う場面は地味に多いかも、、！
https://qiita.com/ktr1211/items/115a9eb037496e3c047d

Reply:
@shingohorie こちら修正しました～！お作法もありがとございます🤲